src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1283,6 +1283,10 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="..\..\tools\install\Extra\DirectX\D3DCompiler_43.dll">
+      <Link>D3DCompiler_43.dll</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
I think using `If Newer` consistently on content like this would speed up builds.